common/lib/wrapper_property.ts

```diff
@@ -201,6 +202,12 @@ export class WrapperProperties {
     30000
   );
 
+  static readonly CLUSTER_TOPOLOGY_HIGH_REFRESH_RATE_MS = new WrapperProperty<number>(
+    "clusterTopologyRefreshRateMs",
```

Comment:
did you want this to be called `clusterTopologyHighRefreshRateMs`. I see in the docs PR that's what this is called